packages/prosekit/CHANGELOG.md

```
@@ -1,5 +1,23 @@
 # prosekit
 
+## 0.22.0-beta.7
+
+### Patch Changes
+
+- [`c9a3e30`](https://github.com/ocavue/prosekit/commit/c9a3e30bad0afe4948ecb325d8a02d8258cc8dee) ![](https://prosekit.dev/b/extensions)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)